pkg/networkpolicy/networkpolicyapi.go

```diff
@@ -60,6 +60,12 @@ func (c *Controller) evaluateAdminEgress(adminNetworkPolices []*npav1alpha1.Admi
 						return rule.Action
 					}
 				}
+
+				for _, domain := range to.DomainNames {
+					if c.dnsCache.containsIP(string(domain), ip) {
+						return rule.Action
+					}
+				}
```

Comment:
@danwinship @tssurya  is not clear to me from the API how this should be implemented, as it says that only allow rules are permitted, does it mean this is correct?

Reply:
ok, then this is ok, if the validation gets relaxed this will do the right thing